docs/about.rst

```diff
@@ -31,3 +31,26 @@ Ralf Gommers contributes to the development of ``meson-python`` as
 part of their job at `Quansight`__.
 
 __ https://www.quansight.com/
+
+
+Packages
+========
+
+``meson-python`` packages are distriburted via `PyPI`_ as Python wheel
+for Python 3.7 and later.
```

Comment:
can we say "all supported Python releases" to avoid having the minimum supported version go out of date here when we drop support for 3.7?

Reply:
also, maybe add "source distribution". Let me propose a full rephrase of this sentence:
```
``meson-python`` packages are distributed via `PyPI`_, as a source distribution and a
single pure Python wheel that works across all supported Python versions.
```

Reply:
I avoided to write "supported" because technically Python 3.7 is already end-of-life but I don't think we are planning to retire support for it. However, I don't think it is important to be overly precise here. I like the proposed wording. I'll update the commit.